tmpl/wiki/hello-world.md

```diff
@@ -16,16 +17,16 @@ This document refers to unikernels in the `tutorial` directory.
 the [Lwt tutorial](tutorial-lwt) first.
 
 **Additional note**: Throughout the tutorial, we'll use `mirage configure -t unix`
-to demonstrate building MirageOS applications.  If you're using a Mac OS X
+to demonstrate building MirageOS applications.  If you're using a macOS
 machine, you should use `mirage configure -t macosx` instead.
 
 ### Step 0: Doing Nothing!
 
 Before we try and do anything complicated, let's do nothing briefly. That is,
-let's build a unikernel that simply starts and then exits -- nothing else. The
+let's build a unikernel that simply starts and then exits—nothing else. The
```

Comment:
Converting `--` to `—` is good, but I think you still need the spaces around the mdash everywhere. At least, `... exits — nothing else` looks better to me than `... exits—nothing else` (which appears more like a hyphen).

Reply:
The [punctuation guide says](http://www.thepunctuationguide.com/em-dash.html) that there's commonly no space before and after the em dash, but I don't have strong feelings about this.